pkg/packagekit/updates.jsx

```diff
@@ -121,6 +121,27 @@ function commaJoin(list) {
     return list.reduce((prev, cur) => [prev, ", ", cur])
 }
 
+class Expander extends React.Component {
+    constructor() {
+        super();
+        this.state = {expanded: false};
+    }
+
+    render() {
+        var title = <a href="#">{this.props.title}</a>;
+        var cls = "expander-caret fa " + (this.state.expanded ? "fa-angle-down" : "fa-angle-right");
+        return (
+            <div>
+                <div className="expander-title">
+                    <hr />
+                    <span onClick={() => this.setState({expanded: !this.state.expanded})} ><i className={cls} />{title}</span>
```

Comment:
I broke it into multiple lines. I put a space before the `>` on all the tags that have lots of properties, makes it a bit easier to see where it ends. If you prefer/if that's our style guide, I'm happy to add an extra commit that removes all those.